src/providers/pronouns/alejo/PronounsAlejoApi.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "providers/pronouns/UserPronouns.hpp"
+
+#include <QJsonObject>
+#include <QString>
+
+#include <mutex>
+#include <optional>
+#include <shared_mutex>
+
+namespace chatterino::pronouns {
+
+class AlejoApi
+{
+private:
+    std::shared_mutex mutex;
+    /** A map from alejo.io ids to human readable representation like theythem -> they/them, other -> other. */
+    std::optional<std::unordered_map<QString, QString>> pronounsFromId =
+        std::nullopt;
+    UserPronouns parse(const QJsonObject &);
```

Comment:
The parameter should have a name (the same as in the implementation - `object`).